nixos/modules/services/hardware/handheld-daemon.nix

```diff
@@ -11,6 +11,11 @@ in
     enable = mkEnableOption "Enable Handheld Daemon";
     package = mkPackageOption pkgs "handheld-daemon" { };
 
+    ui = {
+      enable = mkEnableOption "Handheld Daemon UI";
```

Comment:
Can we have this default to use the same value as `handheld-daemon.enable`?

Reply:
Sure thing